src/clojure/clj_ldap/client.clj

```diff
@@ -114,8 +130,12 @@
 
 (defn- create-ssl-factory
   "Returns a SSLSocketFactory object"
-  ^SSLSocketFactory [{:keys [trust-managers trust-store]}]
-  (let [^SSLUtil ssl-util (create-ssl-util trust-managers trust-store)]
+  ^SSLSocketFactory [{:keys [trust-managers trust-store cipher-suites ssl-protocols]}]
+  (let [^SSLUtil ssl-util (create-ssl-util trust-managers trust-store)] 
+    (when (not-nil? cipher-suites)
+      (.setEnabledCipherSuites cipher-suites))
```

Comment:
```suggestion
      (.setEnabledCipherSuites ssl-util cipher-suites))
```

Reply:
We are calling the `setEnabledCipherSuites` function in the SSLUtil class, so need to provide the instance.

Reply:
I think we may have to convert the last argument into the right type for the java interface.  Trying to figure out the best way to do that.

Reply:
Actually I think the type should work.